metatron-app/metatron-discovery
# #3739 single date picker timefilter

### Description
If you set granularity , you need a filter that can select a single date.

**Related Issue** : #3739 


### How Has This Been Tested?

1. Open the filter creation popup.
2. Select the time field, select granularity, and make sure that there is a single time type in Filter Type.
3. After selecting a single time type, check if it is displayed normally on the panel.
4. Register the filter widget and check if it is displayed normally in the widget.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
